### PR TITLE
Migration guide feedback

### PR DESCRIPTION
**Pull request by Witoso**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Docs: Update the migration guide.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 1]
```diff
@@ -94,11 +94,11 @@ ClassicEditor
 	} );
 ```
 
-It may seem strange to show the webpack configuration in an example of the old installation methods, but for many it was a necessary part of the setup to handle translations, CSS, and SVG files. This setup could be even more complex if you wanted to use TypeScript.
```
Comment (Witoso) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:97]: leaving out "for many" as many may not know they are part of many.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 2]
```diff
@@ -94,11 +94,11 @@ ClassicEditor
...
+In the new installation methods we have reduced the number of possible paths to just two: **npm packages and browser builds**. Unlike before, both methods no longer require you to add dozens of individual packages or JavaScript bundles to get the editor up and running. Instead, you can import the editor and all our open source plugins from the `ckeditor5` package and the premium features from `ckeditor5-premium-features`. You also do not need to worry about a specific webpack or Vite configurations, as the new installation methods are designed to work out-of-the-box with any modern bundler or JavaScript meta-framework like Next.js.
```
Comment (Witoso) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:101]: Grammar + adding some specific meta framework.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 3]
```diff
@@ -130,7 +130,7 @@ ClassicEditor
 
 ### Browser builds
 
-The browser builds are a great way to use CKEditor&nbsp;5 if you do not use a module bundler. The browser builds are available as JavaScript modules and can be loaded directly in the browser using the `<script type="module">` tag.
+The browser builds are a great way to use CKEditor&nbsp;5 if you do not want to build JavaScript with a module bundler. The browser builds are available as JavaScript modules and can be loaded directly in the browser using the `<script type="module">` tag.
```
Comment (Witoso) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:133]: adding explicit building action.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 4]
```diff
@@ -209,7 +209,7 @@ Here is a visual comparison of the features available in the new npm and CDN bui
 			<td>✅</td>
 		</tr>
 		<tr>
-			<td>Works with any bundler or meta-framework</td>
+			<td>Can be used with any modern bundler</td>
```
Comment (Witoso) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:212]: leaving out meta-framework, as it's a very JS tech speak.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 5]
```diff
@@ -317,15 +317,15 @@ What we will sunset on this date:
...
-If you maintain custom plugins for CKEditor&nbsp;5, you need to update them first.
+First, proceed with migrating your project, depending on the old installation method you are using.
 
-* {@link updating/nim-migration/custom-plugins Migrating custom plugins}
+* {@link updating/nim-migration/predefined-builds Migrating from predefined builds}.
+* {@link updating/nim-migration/customized-builds Migrating from customized builds}.
+* {@link updating/nim-migration/dll-builds Migrating from DLL builds}.
 
-After you have updated your custom plugins, you can proceed with migrating your project, depending on the old installation method you are using.
+Second, if you maintain any CKEditor&nbsp;5 custom plugins **and publish them to npm** for you need to update them as well:
```
Comment (Witoso) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:326]: I changed the order as I consider custom libs on npm, a secondary action not the first one.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md [hunk 6]
```diff
@@ -317,15 +317,15 @@ What we will sunset on this date:
 
 To migrate your project to the new installation methods, you can follow the instructions below.
 
-If you maintain custom plugins for CKEditor&nbsp;5, you need to update them first.
+First, proceed with migrating your project, depending on the old installation method you are using.
```
Comment (filipsobol) [inline docs/updating/nim-migration/migration-to-new-installation-methods.md:320]: Plugins must be migrated first

File: docs/updating/nim-migration/migration-to-new-installation-methods.md (hunk 5 above)
Comment (filipsobol) [reply docs/updating/nim-migration/migration-to-new-installation-methods.md:326]: It's not necessary to publish them to npm, but just having them as a separate npm package, for example in a monorepo.

File: docs/updating/nim-migration/migration-to-new-installation-methods.md (hunk 6 above)
Comment (Witoso) [reply docs/updating/nim-migration/migration-to-new-installation-methods.md]: Ok, you convinced me :D

Comment (Witoso) [reply docs/updating/nim-migration/migration-to-new-installation-methods.md]: Ok, you convinced me :D

**Review by godai78 — APPROVED** [review]

**Pull request merged**